geoips/commandline/log_setup.py

```diff
@@ -16,6 +16,24 @@
 import sys
 
 
+def log_with_emphasis(LOG, messages):
+    """Log the given message with emphasis provided a certain log type.
```

Comment:
I think the person passing in the lines should just be responsible for making sure each line is the length they want it...

Pass in multiple lines of the length you want them printed to the screen.  Normal calls to LOG or print don't modify the length of your lines, I think we can safely assume people pass line lengths they want printed.

Reply:
I agree with @mindyls  here - that the convention is to make the user handle the string length for logging functions so we probably should stick with that and not auto-wrap the logged string.

I'm concerned because this function "wraps" the log string in `*`s which means that if you have a long string you could get:

```
***********************************************
** Testing strings for various purposes can be quite intriguing, as it involves exploring different functionalities and edge cases. Testing strings for various purposes can be quite intriguing, as it involves exploring different functionalities and edge cases. Testing strings for various purposes can be quite intriguing, as it involves exploring different functionalities and edge cases. **
***********************************************
```

which wraps like this on my terminal:

![image](https://github.com/NRLMMD-GEOIPS/geoips/assets/9918036/ff43fe9c-3ab7-4c79-91df-964026a7ecf7)

which is a bit confusing. I feel like this:

```
***********************************************
** A very long string just for testing this. **
** A very long string just for testing this. **
** A very long string just for testing this. **
***********************************************
```
is a lot more readable, but requires us to wrap the string which breaks convention on logging not handling the wrapping of the output.

At this point I'm wondering if simply pre-pending the logged line with `***` might be cleaner and more in convention? What is the purpose of this new logging level?

eg. the output would be:

```
*** Testing strings for various purposes can be quite intriguing, as it involves exploring different functionalities and edge cases. Testing strings for various purposes can be quite intriguing, as it involves exploring different functionalities and edge cases. Testing strings for various purposes can be quite intriguing, as it involves exploring different functionalities and edge cases.
```

What do you think @mindyls ?

Reply:
Just an idea - would this be better with colorized output? (eg. we could do a different color for emphasis?)

Reply:
The purpose of this new logging level is that we wanted to consolidate all of the "*********" scattered throughout the code, so wanted to replace it with a new log level.

The only requirements from my perspective are to replicate what those "*****"s did before:
* pass in a list of strings
* Put "*"*80 before the strings
* print the strings one at a time
* Put "*"*80 after the strings.